### PR TITLE
Perimeter

### PR DESCRIPTION
**Pull request by tryfon-mw**
Added perimeter computation in the compute_volume function as well as a test in test_helpers.py.

### CODE REVIEW

File: test/helpers.py [hunk 1]
```diff
@@ -38,6 +38,10 @@ def compute_volume(points, cells):
                 ]).T
             mesh = voropy.mesh_tri.MeshTri(*prune_nodes(points, split_cells))
             vol += math.fsum(mesh.cell_volumes)
+    else:
+        assert 'line' in cells
+        segs = numpy.diff(points[cells['line']], axis=1).squeeze()
+        vol = numpy.sum(LA.norm(segs, axis=1))
```
Comment (nschloe) [inline test/helpers.py]: No need for scipy here; `einsum('i...,i...->...', segs, segs)` should be faster anyway.

Comment (tryfon-mw) [reply test/helpers.py]: Made the change using einsum but it requires a sqrt for each segment then a final sum over all segments. But scipy is removed. See commit b1d3593a383d2c8799e36c14597d71a9a776ac10

File: test/test_helpers.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+import pygmsh
+from helpers import compute_volume
+
+
+def test():
+    geom = pygmsh.built_in.Geometry()
+    geom.add_circle([0, 0, 0], 1, 0.1, make_surface=False)
+    points, cells, _, _, _ = pygmsh.generate_mesh(geom)
+    assert np.abs((compute_volume(points, cells) - 2 * np.pi) / np.pi / 2) < 1e-2
```
Comment (nschloe) [inline test/test_helpers.py]: Just a small nitpick: You're listing the reference value twice here, perhaps more readable would be
```
ref = 2 * np.pi
assert np.abs(compute_volume(points, cells) - ref) < 1e-2 * ref
```

**Comment by nschloe** [comment]
LGTM.

**Pull request merged**